Mob/mob.gd

```diff
@@ -0,0 +1,11 @@
+extends CharacterBody2D
+
+var player 
+
+func _ready():
+	player = get_node("/root/Game/Player")
+
+func _physics_process(delta):
+	var direction = global_position.direction_to(player.global_position)
+	velocity = direction * 300.0
```

Comment:
[Snurrigt1](https://github.com/Snurrigt1) commented on the speed. This is something that will likely need to be tweeked over time. As such it should probably be made easily configured.